Exporter/src/main/java/uk/ac/ebi/biostd/exporter/persistence/dao/SubmissionDao.java

```diff
@@ -35,6 +35,11 @@
                 queries.getUpdatedSubmissionsQuery(), singletonMap("sync_time", syncTime), submissionMapper);
     }
 
+    public Submission getSubmissionByAccNo(String accNo) {
+        return template.queryForObject(queries.getSubmissionsQueryByAccNo(), singletonMap("accno", accNo),
```

Comment:
We should get to an agreed standard about how we wrap lines, for example in the first parenthesis, something like this:
```
method(anotherMethodWithReallyLongName(
        paramWithReallyLongName1, paramWithReallyLongName2)
```
Not saying this is the one but we should have something consistent.